cmd/fleetctl/query.go

```diff
@@ -92,15 +92,18 @@ func queryCommand() cli.Command {
 				select {
 				case hostResult := <-res.Results():
 					out := resultOutput{hostResult.Host.HostName, hostResult.Rows}
+					s.Stop()
 					if err := json.NewEncoder(os.Stdout).Encode(out); err != nil {
 						fmt.Fprintf(os.Stderr, "Error writing output: %s\n", err)
 					}
+					s.Start()
 
 				case err := <-res.Errors():
 					fmt.Fprintf(os.Stderr, "Error talking to server: %s\n", err.Error())
 
 				case <-tick.C:
 					// Print status message to stderr
+					statusfmt := " %.f%% responded (%.f%% online) | %d/%d targeted hosts (%d/%d online)"
```

Comment:
Can we `fmt.Sprintf` this status string once, then output it as appropriate lower down? I'd prefer not to see all the values plugged in multiple times (on line 122 and line 125).